plugin/kubernetes/kubernetes.go

```diff
@@ -294,6 +302,29 @@ func (k *Kubernetes) Records(state request.Request, exact bool) ([]msg.Service,
 	return services, err
 }
 
+// ServiceFQDN returns the k8s cluster dns spec service FQDN for the service (or endpoint) object
+func ServiceFQDN(obj meta.Object, zone string) string {
+	return dnsutil.Join(append([]string{}, obj.GetName(), obj.GetNamespace(), Svc, zone))
+}
+
+// PodFQDN returns the k8s cluster dns spec FQDN for the pod
```

Comment:
not ending dot and dns -> DNS (more comments have not period on the end)

Reply:
done